hnn_core/gui/gui.py

```diff
@@ -836,21 +837,55 @@ def _simulate_viz_action(self, action_name, *args, **kwargs):
         action(*args, **kwargs)
 
 
+def _prepare_upload_file_from_local(path):
+    with open(path, 'rb') as file:
+        content = memoryview(file.read())
+    last_modified = datetime.fromtimestamp(path.stat().st_mtime)
+
+    upload_structure = [{
+        'name': path.name,
```

Comment:
Is there any significance to using the full path for `name` in the local file vs. that last characters after a `'/'` in the URL upload? Is it just metadata so it doesn't matter that much?

Reply:
The name attribute of a Path object is the filename. Though this reminds me that all to add a conversion to Path in this function for if a string path is passed as an argument.